deployment/v3/mosip/mosipcertmanager/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Installs sample print service
+## Usage: ./restart.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+
+NS=mosipcertmanager
+CHART_VERSION=0.0.1-develop
+
+echo Create $NS namespace
+kubectl create ns $NS 
+
+function installing_mosipcertmanager() {
+  echo Istio label
+  kubectl label ns $NS istio-injection=disabled --overwrite
+  helm repo update
+
+  echo Copy configmaps
+  sed -i 's/\r$//' copy_cm.sh
+  kubectl -n $NS delete --ignore-not-found=true cm s3
```

Comment:
Please remove this line 23

Reply:
Taken Care & hence resolving Conversation.